src/main/webapp/availability.js

```diff
@@ -33,7 +33,13 @@ function toggleTile(tile) {
 }
 
 function loadAvailabilityTable(tableDiv, timezoneOffset) {
-  fetch(`/availabilityTable.jsp?timeZoneOffset=${timezoneOffset}`)
+  if (page < 0) {
```

Comment:
You've got guard logic for this in `goForward` and `goBack`, so this should not be necessary

Reply:
Done